### PR TITLE
Adds flake8-isort 2.4

### LINKED ISSUE #6: Use flake8-isort

### PR DESCRIPTION
**Pull request by vsham20**
Fixes #6.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,2 @@
+flake8-isort==2.4
```
Comment (marco-c) [inline requirements.txt]: Can you create a separate `requirements-test.txt` file with flake8-isort?
The requirements.txt lists packages needed to use the project, `requirements-test.txt` lists the packages needed to work on the project.

File: crawler.py [hunk 2]
```diff
@@ -1,10 +1,13 @@
 import json
 import os
-import time
 import random
+import time
 import traceback
+
 from selenium import webdriver
-from selenium.common.exceptions import NoAlertPresentException, NoSuchWindowException, TimeoutException
+from selenium.common.exceptions import (NoAlertPresentException,
```
Comment (marco-c) [inline crawler.py]: Let's configure flake8-isort so that we don't need to use multiple lines in cases like this

File: requirements.txt (hunk 1 above)
Comment (vsham20) [reply requirements.txt]: Sure.

File: crawler.py (hunk 2 above)
Comment (vsham20) [reply crawler.py]: """
from selenium.common.exceptions import NoAlertPresentException
from selenium.common.exceptions import NoSuchWindowException
from selenium.common.exceptions import TimeoutException
"""
Is this the expected behaviour in this case?To achieve this a new isort cfg file will be added with all the configuration inside it.

Comment (marco-c) [reply crawler.py]: No sorry, to be clear, I meant something like before, where we have all the imports on the same line

**Pull request merged**